From 2caa28e0c67260eb1ecb18444ecbf3b624ca11ed Mon Sep 17 00:00:00 2001
From: Brad Chou <bradc@hyvedesignsolutions.com>
Date: Wed, 18 Sep 2019 16:00:07 +0800
Subject: [PATCH] ARM: dts: aspeed: hs2500: Enable ADC channel 0 -7 There are 8
 ADC channels connected to HSBUV. ADC0 is 2/3 of 3.3V but max reading is 1.8V
 ADC1 is 1/2 of 3.3V ADC2 is 1/3 of 3.3V ADC3 is 1/4 of 3.3V ADC4 - ADC7 are
 adjustable from 0V - 1.8V

---
 arch/arm/boot/dts/aspeed-bmc-opp-hs2500.dts | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed-bmc-opp-hs2500.dts b/arch/arm/boot/dts/aspeed-bmc-opp-hs2500.dts
index 0ebfdf9..a0f40b5 100644
--- a/arch/arm/boot/dts/aspeed-bmc-opp-hs2500.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-opp-hs2500.dts
@@ -132,9 +132,10 @@
 		};
 	};
 
-	iio-hwmon-battery {
+	iio-hwmon {
 		compatible = "iio-hwmon";
-		io-channels = <&adc 12>;
+		io-channels = <&adc 0>, <&adc 1>, <&adc 2>, <&adc 3>,
+			<&adc 4>, <&adc 5>, <&adc 6>, <&adc 7>;
 	};
 };
 
